truss/remote/baseten/api.py

```diff
@@ -109,6 +109,9 @@ def create_model_from_truss(
         is_trusted: bool,
         deployment_name: Optional[str] = None,
         origin: Optional[b10_types.ModelOrigin] = None,
+        chain_environment: Optional[str] = None,
```

Comment:
we should make sure the django mutation to accept these inputs is merged (and makes it all the way to production) before merging this PR